releasetool/commands/tag/ruby.py

```diff
@@ -157,4 +157,10 @@ def tag(ctx: TagContext = None) -> TagContext:
     if ctx.interactive:
         click.secho(f"\\o/ All done!", fg="magenta")
 
+    branch = ctx.release_pr["head"]["ref"]
+    ctx.github_release = ctx.github.delete_branch(
```

Comment:
```suggestion
    ctx.github.delete_branch(
```
I don't think there's a need to set `ctx.github_release` to`None`?

Reply:
:man_facepalming: